alpine-nix-rails-nginx/example-rails-template.rb

```diff
@@ -26,16 +26,42 @@
 web: bundle exec puma --quiet --config config/puma.rb
 PROCFILE
 
+file 'config/postgres.sh', <<-'POSTGRES'
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
```

Comment:
What if we move the `set` and the `IFS` to a shared file we can include, instead of having to repeat ourselves over and over again? I remember passing by this a lot in this repo.